Sources/HipMobileUI/App.xaml

```diff
@@ -22,10 +22,10 @@
              xmlns:converters1="clr-namespace:PaderbornUniversity.SILab.Hip.Mobile.UI.Converters;assembly=HipMobileUI">
     <Application.Resources>
         <ResourceDictionary>
-            <Color x:Key="PrimaryColor">#3F51B5</Color>
-            <Color x:Key="PrimaryDarkColor">#303F9F</Color>
-            <Color x:Key="AccentColor">#FF9800</Color>
-            <Color x:Key="AccentDarkColor">#E65100</Color>
+            <Color x:Key="PrimaryColor">#7FACFF</Color>
+            <Color x:Key="PrimaryDarkColor">#0149D1</Color>
+            <Color x:Key="AccentColor">#FFE57F</Color>
+            <Color x:Key="AccentDarkColor">#FFCC00</Color>
             <Color x:Key="LightGrayColor">#05000000</Color>
             <Color x:Key="MediumGrayColor">#15000000</Color>
             <Color x:Key="DarkGrayColor">#66000000</Color>
```

Comment:
There are no shades of gray defined in the Corporate Design Manual. Do you think we can keep these or should we rework them as well?

Reply:
you mean the grey as a background color in corporate manual? But that's no part of the design, I think keep the current grey should be fine